resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/DeletedDiskChecker.scala

```diff
@@ -0,0 +1,37 @@
+package com.broadinstitute.dsp
+package resourceValidator
+
+import cats.effect.{Concurrent, Timer}
+import cats.implicits._
+import cats.mtl.ApplicativeAsk
+import io.chrisdavenport.log4cats.Logger
+import org.broadinstitute.dsde.workbench.google2.GoogleDiskService
+import org.broadinstitute.dsde.workbench.model.TraceId
+
+// Implements CheckRunner[F[_], A]
+object DeletedDiskChecker {
+  def impl[F[_]: Timer](
+    dbReader: DbReader[F],
+    deps: DeletedDiskCheckerDeps[F]
+  )(implicit F: Concurrent[F], logger: Logger[F], ev: ApplicativeAsk[F, TraceId]): CheckRunner[F, Disk] =
+    new CheckRunner[F, Disk] {
+      override def configs = CheckRunnerConfigs("deleted-disks", true)
+      override def dependencies: CheckRunnerDeps[F] = deps.checkRunnerDeps
+      override def aToScan: fs2.Stream[F, Disk] = dbReader.getDeletedDisks
```

Comment:
I'm guessing `a` refers to a type here. I wonder if it'd be more understandable to rename the corresponding methods as `resourcesToScan` and `checkResource`.